src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.logic.commands;
+
+/**
+ * Sorts all coins in Coin List by alphabetical order and displays sorted list to user
+ */
+//@@author neilish3re
+public class SortCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "order";
+    public static final String COMMAND_ALIAS = "o";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all the coins the user has input into the Coin Book as a list"
```

Comment:
Usage message should include description of parameters

Also update User Guide